applications/stepswitch/src/stepswitch_bridge.erl

```diff
@@ -136,11 +136,25 @@ handle_cast({'bridge_result', _Props}, #state{response_queue='undefined'}=State)
 handle_cast({'bridge_result', Props}, #state{response_queue=ResponseQ}=State) ->
     wapi_offnet_resource:publish_resp(ResponseQ, Props),
     {'stop', 'normal', State};
-handle_cast({'bridged', CallId}, #state{timeout='undefined'}=State) ->
-    lager:debug("channel bridged to ~s", [CallId]),
+handle_cast({'bridged', CallId, BridgedTo}, #state{timeout='undefined'
```

Comment:
Why is stepswitch publishing an event here? Why not just listen for `CHANNEL_BRIDGE` in the cf_resources process?